README.md

```diff
@@ -1,9 +1,45 @@
 # PHP Releases Action
-Docker container action to consume the PHP Releases API for use within GitHub Actions.
+Generates an array of supported PHP releases for use within a GitHub Actions matrix.
 
-## Requirements
+## Usage
+Within your workflow `jobs` block, add the following job _before_ your matrix job:
+```yaml
+  output_releases:
+    name: Generate PHP Releases Array
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Current Releases
+        uses: tighten/phpreleases-action@v1
+        id: releases
+```
+If you need to add specific versions to the matrix statically (to ensure they're included regardless of current support), add them as a comma separated list with the `releases` key:
+```yaml
+  output_releases:
+    name: Generate PHP Releases Array
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Current Releases
+        uses: tighten/phpreleases-action@v1
+        id: releases
+        with:
+          releases: '7.4, 7.3'
+```
```

Comment:
Similar to above about inlining the usage details. This is just an opinion but I think finishing the usage details before going into an outlier would be better.

```suggestion
 ### Create a matrix from the return value
```yaml
  current_php_releases:
    runs-on: ubuntu-latest
    # The matrix cannot be built before the job has finished.
    needs: output_releases
    strategy:
      matrix:
        # GitHub Actions expression to get the return value.
        php: ${{ fromJSON(needs.output_releases.outputs.range) }}
    name: PHP ${{ matrix.php }}
```


### Add PHP versions that are not included by default

```yaml
  # This job will need to run before the job that defines the matrix.
  output_releases:
    name: Generate PHP Releases Array
	  # Requires a machine that can execute bash and make http requests.
    runs-on: ubuntu-latest
    steps:
      - name: Fetch Current Releases
        uses: tighten/phpreleases-action@v1
        id: releases
        with:
          # Comma delimited string of all versions that should be included in the matrix.
          releases: '7.4, 7.3'
```
```